syntax = "proto2";
package oneflow;

enum MemoryFormat {
  kContiguous = 0;
  kChannelsLast = 1;
  MemoryFormat_Max = 2;
  kPreserve = 3;

};
